<launch>

    <node pkg="pulson_ros" name="tdma_node" type="tdma_node" output="screen">
        <param name="pulse_integration_index" value="5" />
        <param name="node_id" value="100" />
        <param name="slot_map_file" value="$(find pulson_ros)/config/motion_tests.yaml" />
	<param name="ip_address" value="0" />
	<param name="serial_dev" value="" />
        <param name="dev_path" value="" />
    </node>

</launch>
